src/index.js

```diff
@@ -156,17 +156,21 @@ export function app(state, actions, view, container) {
 
   function updateAttribute(element, name, value, oldValue, isSvg) {
     if (name === "key") {
-    } else if (name === "style") {
-      for (var i in clone(oldValue, value)) {
-        var style = value == null || value[i] == null ? "" : value[i]
-        if (i[0] === "-") {
-          element[name].setProperty(i, style)
+    } else {
+      if (name === "style") {
+        if (typeof value === "string") {
+          element[name].cssText = value
```

Comment:
What if we also check if `value == null`? 🤔

Reply:
maybe just `typeof value !== "object"` instead of `typeof value === "string" || value == null`?

Reply:
or you are talking about `cssText = value == null ? '' : value`?
This could work too, but it seems it changes nothing